src/sass/_permission_groups.scss

```diff
@@ -0,0 +1,125 @@
+.permission-groups {
+  .groups-table {
+    .name {
+      width: 15%;
+    }
+
+    .description {
+      width: 45%;
+
+      span {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+
+        @supports (-webkit-line-clamp: 2) {
+          -webkit-box-orient: vertical;
+          display: -webkit-box;
+          -webkit-line-clamp: 2;
+          overflow: hidden;
+          text-overflow: ellipsis;
+          white-space: initial;
+          word-wrap: break-word;
+        }
+      }
```

Comment:
Here we again have the two line clamping. Can we remove it and rely on 1 line truncation with the vanilla u-truncate class?

If not, let's reuse this i.e. with an `@include` and not copy it in too many places.

Reply:
Let's create the mixin for now, we can have a separate discussion with piper post the merging or this PR about this